app/javascript/chat/components/ChannelButton.jsx

```diff
@@ -0,0 +1,123 @@
+import { h, Component, createRef } from 'preact';
+import PropTypes from 'prop-types';
+
+class ChannelButton extends Component {
+  buttonRef = createRef();
+
+  componentDidMount() {
+    const { isActiveChannel = false } = this.props;
+
+    if (isActiveChannel) {
+      this.buttonRef.current.click();
+    }
+  }
+
+  renderChannelImage = () => {
+    const { channel, newMessagesIndicator, discoverableChannel } = this.props;
+
+    return (
+      <span
```

Comment:
refactor (non-blocking): Is the span necessary or can all these data-attributes be on the `<img />`?

Reply:
This span is necessary for another styling for the message indicator.